code/game/gamemodes/wizard/artefact.dm

```diff
@@ -545,14 +545,18 @@
 		var/area/A = get_area(src)
 		to_chat(victim, "<span class='notice'>You feel a dark presence from [A.name]</span>")
 
+/obj/item/voodoo/suicide_act(mob/living/carbon/user)
+    user.visible_message("<span class='suicide'>[user] links the voodoo doll to themself and sits on it, infinitely crushing themself! It looks like [user.p_theyre()] trying to commit suicide!</span>")
+    user.gib()
+    return(BRUTELOSS)

```

Comment:
why bruteloss if they're gibbed?

Reply:
if for some reason it doesn't go off when a snowflake mob gets added or something, we don't want a mob alive with the suicide on.

Reply:
I think bruteloss is necessary like arm said. Just in case a mob doesn't gib after doing the suicide. Game breakery prevention!

Reply:
You have to return some harm type anyway

Reply:
Not if you return `MANUAL_SUICIDE`, but this bruteloss is proper here anyway.